Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM php:8-apache  
+RUN set -ex && bash -c "set -euxo pipefail && install <(curl -fsSLo - "$(curl -sLI -o /dev/null -w '%{url_effective}' "https://github.com/tianon/gosu/releases/latest" | sed 's/tag/download/')/gosu-amd64") /usr/local/bin/gosu" && gosu --version
```

Comment:
tl:dr it was an attempt to drop all root privs for the apache processes, but still do root necessary things ( i.e. chown'ing any files to the www-data user ) and I will remove it.

So, initially I was going to try and implement gosu like how the [mysql library container does](https://github.com/docker-library/mysql/blob/c506174eab8ae160f56483e8d72410f8f1e1470f/8.0/docker-entrypoint.sh#L379) to drop permissions to the www-data user when running the command, but forgot to implement it before actually doing this PR :sweat_smile: . 

After looking into it further it looks like I will have to launch the initial apache2 process as root, but then everything else (besides the first process) should be running as a non-root user.

### backstory

My goal was to do something like this:

![image](https://user-images.githubusercontent.com/10230166/132953409-951846a0-3008-45f8-addf-dc6853145188.png)

When I was trying initially to run everything [as the www-data user](https://github.com/elreydetoda/hashtopolis-server/blob/3b48f1c9b562dc466ccefbd68ca2d6dfefe6a8e2/Dockerfile#L9) it worked up untill I tried to do a bind mount so I could persist the config file as well as the 2 directories. So, then I had to keep the root permissions so I could chown the root files ( which is the default permissions of files that are bind mounted ). Which then spurred the thought of using gosu, but when I use gosu it can't get a proper tty or something because when it tries to read the logs at /var/logs/apache2/* it fails to open them.

Reply:
[done](https://github.com/elreydetoda/hashtopolis-server/commit/bf50b4fccf3f855df6386f49fe9d8deda28425cc)

Reply:
Thanks! Never heard of gosu before, but gotta love apache :grin: